src/css/object/project/features.scss

```
@@ -0,0 +1,46 @@
+@use '../../object/utility/utility.scss';
+
+.p-features {
+  padding: 110px 0px;
```

基本的に全てpxではなくremでやってみましょうか🙏
```suggestion
  padding: 11rem 0;
```

---

pxをすべてremに変換しました。